recipe/meta.yaml

```diff
@@ -40,35 +40,35 @@ requirements:
     - {{ posix }}zip                 # [win]
   host:
     - r-base
-    - libcurl                        # [not win]
+    - libcurl
     # Test below (connecting to rest.kegg.jp) triggers upstream issue described in
     # https://github.com/curl/curl/issues/13389 and https://github.com/openssl/openssl/issues/11378
     # Building against libcurl built against openssl<3.2 allows to circumvent that issue if needed.
     # TODO: Remove explicit dependency when workaround is not needed anymore.
-    - openssl <3.2.0a0               # [not win]
+    - openssl <3.2.0a0  # [unix and r_base != "4.4"]
   run:
     - r-base
-    - {{ native }}gcc-libs           # [win]
-    - {{ native }}libwinpthread-git  # [win]
-    - {{ native }}zlib               # [win]
 
 test:
   requires:
+{% if version != "5.2.1" or build != 1 and (win or r_base == "4.4") %}  # temporarily disabled for migration
     - r-httpuv
     - r-testthat
     # NB: r-webutils creates a circular dependency; it should be temporarily disabled during migrations
     - r-webutils
     # TODO: See above. Remove explicit dependency when is workaround not needed anymore.
-    - openssl <3.2.0a0
+    - openssl <3.2.0a0  # [unix and r_base != "4.4"]
+{% endif %}
   source_files:
     - tests/
     - tests/testthat
   commands:
-    - $R -e "library('curl')"                                                                 # [not win]
-    - $R -e "testthat::test_file('tests/testthat.R', stop_on_failure=TRUE)"                   # [not win]
-    - $R -e "invisible(curl::curl_fetch_memory('http://rest.kegg.jp/conv/ncbi-geneid/dme'))"  # [not win]
-    - "\"%R%\" -e \"library('curl')\""                                                 # [win]
-    - "\"%R%\" -e \"testthat::test_file('tests/testthat.R')\""  # [win]
+    - R -e "library('curl')"
+{% if version != "5.2.1" or build != 1 and (win or r_base == "4.4") %}  # temporarily disabled for migration
+    - R -e "testthat::test_file('tests/testthat.R', stop_on_failure=TRUE)"
+    # skip for unix and r_base == "4.4" because r-base=4.4 is built against newer libcurl with openssl>=3.3 dep.
+    - R -e "invisible(curl::curl_fetch_memory('http://rest.kegg.jp/conv/ncbi-geneid/dme'))"  # [not unix or r_base != "4.4"]
```

Comment:
The comment from https://github.com/conda-forge/r-curl-feedstock/pull/31#discussion_r1667754793 only applies for `r-base=4.3` since all our `r-base=4.4` will indirectly pull in `openssl>=3.3`.